hack/ci/cloud-init/controller.yaml.tpl

```diff
@@ -34,8 +34,10 @@
     # Glance
     ENABLED_SERVICES+=,g-api
 
-    # Neutron
-    ENABLED_SERVICES+=,neutron-api,neutron-agent,neutron-dhcp,neutron-l3
+    # Octavia-Neutron
+    Q_AGENT=linuxbridge
+    ENABLED_SERVICES+=,neutron-api,neutron-agent,neutron-dhcp,neutron-l3,neutron-trunk
```

Comment:
curious we didn't have neutron-trunk before, so this means no trunk test before?

Reply:
There was e2e trunk test before.

Reply:
Yes, it's a long-standing oversight.